packages/IMAPClient-Core.package/ICFolder.class/instance/checkTopicality.st

```diff
@@ -1,4 +1,4 @@
-updateEmails
+validating
```

Comment:
I would not expect this method to be in validating. I would expect in validation only methods which return true or false.